services/core-api/app/api/now_applications/resources/now_application_resource.py

```diff
@@ -42,9 +43,13 @@ def get(self, application_guid):
         if now_application_identity.now_application_id and not original:
             application = now_application_identity.now_application
             application.imported_to_core = True
+            application.is_historic = ApplicationsView.query.filter_by(
```

Comment:
This can be moved out of the if/else block and just done once.

Reply:
oh good point